src/Dfe.PlanTech.Web/Controllers/QuestionsController.cs

```diff
@@ -1,20 +1,27 @@
-using Dfe.PlanTech.Application.Submission.Commands;
+using Dfe.PlanTech.Application.Questionnaire.Queries;
+using Dfe.PlanTech.Application.Submission.Interface;
 using Dfe.PlanTech.Application.Submission.Interfaces;
 using Dfe.PlanTech.Domain.Questionnaire.Constants;
 using Dfe.PlanTech.Domain.Questionnaire.Models;
+using Dfe.PlanTech.Web.Helpers;
 using Dfe.PlanTech.Web.Models;
 using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
 
 namespace Dfe.PlanTech.Web.Controllers;
 
 [Authorize]
-[Route("/question")]
 public class QuestionsController : BaseController<QuestionsController>
 {
-    public QuestionsController(ILogger<QuestionsController> logger) : base(logger) { }
+    private readonly GetQuestionQuery _getQuestionQuery;
+    public QuestionsController(ILogger<QuestionsController> logger, GetQuestionQuery getQuestionQuery) : base(logger) 
+    {
+        _getQuestionQuery = getQuestionQuery;
+    }
 
-    [HttpGet("{id?}")]
+    [HttpGet("/question/{id?}")]
+    [Route("{SectionSlug}/{question}/{id?}", Name = "SectionQuestionAnswerWithId")]
+    [Route("{SectionSlug}/{question}", Name = "SectionQuestionAnswer")]
```

Comment:
1. Do not believe the `HttpGet` attribute on line 22 is now necessary
2. Are both these `Route` attributes necessary? They are identicaly
3. `HttpGet` would be better than `Route` here, unless there is any need for `POST` or `DELETE` etc.

Reply:
Changed to leave only one HTTPGet